lib/robots/dor_repo/speech_to_text/stt_create.rb

```diff
@@ -4,17 +4,54 @@ module Robots
   module DorRepo
     module SpeechToText
       # Call the speech to text service to generate text
-      class SttCreate < LyberCore::Robot
+      class SttCreate < Dor::TextExtraction::Robot
         def initialize
           super('speechToTextWF', 'stt-create')
         end
 
         # available from LyberCore::Robot: druid, bare_druid, workflow_service, object_client, cocina_object, logger
         def perform_work
-          # TODO: do the caption creation by calling out to Whisper
+          # start the caption creation by sending an SQS message to the caption creation service
+
+          send_sqs_message
 
           # Leave this step running until the Whisper monitoring job marks it as complete
           LyberCore::ReturnState.new(status: :noop, note: 'Initiated SpeechToText.')
+        rescue StandardError => e
+          Honeybadger.notify('Problem sending SQS Message to AWS for SpeechToText', context: { druid:, error: e })
+          raise "Error sending SQS message: #{e.message}"
+        end
+
+        private
+
+        def send_sqs_message
+          message_body = {
+            id: SecureRandom.uuid,
+            druid:,
+            media:
+          }.merge(whisper_options).to_json
+
+          # Send the message to the SQS queue
+          aws_provider.sqs.send_message({
+                                          queue_url: aws_provider.sqs_todo_queue_url,
+                                          message_body:
+                                        })
+
+          logger.info("Sent SQS message for druid #{druid} to queue #{aws_provider.sqs_todo_queue_url}")
```

Comment:
might be nice to log job id also, while we're at it

Reply:
https://github.com/sul-dlss/common-accessioning/issues/1369

Reply:
Done in #1370